src/services/users/reset-password/reset-password.test.js

```diff
@@ -0,0 +1,24 @@
+const chai = require('chai')
+const verifyResetToken = require('./verifyPasswordResetToken')
+const generateAccessToken = require('../generate-access-token')
+
+const testEmail = 'testemail@gmail.com'
+
+        
+const { expect } = chai
+
+describe('verifyPasswordResetToken', () => {
+	it('should throw an error if the token is invalid', () => {
+		const token = 'invalid-token'
+		expect(() => verifyResetToken(token)).to.throw()
+	})
+})
+describe('verifyPasswordResetToken', () => { 
+	it('should return the decoded email if the token is valid', () => {
+		const token = generateAccessToken(
```

Comment:
we should try to not couple the functions in the tests so much since this function does not have a test on its own. Using the jwt library directly is fine

Reply:
@itzgeebee i think this is yet to be attended to

Reply:
@jocrah Yeah. I can't seem to find why it's still here to be honest

Reply:
@jocrah Also, when I check the file, I can't seem to find it there too. Kindly confirm if it's same on your end

Reply:
@jocrah Why it's not outdated even though I have changed the code

Reply:
@itzgeebee so I'm  not sure I understand the issue yet because other parts of the same file have been changed and are reflecting. 

Are you saying you're not using the `generateAccessToken` function anymore?

Reply:
Don't worry about it @jocrah

Reply:
> Don't worry about it @jocrah

Not sure I got this @itzgeebee. Still confused about this particular convo

Reply:
It's no longer an issue @jocrah 

> @itzgeebee i think this is yet to be attended to

It was a follow up  to this but it's not an issue any more

Reply:
@itzgeebee yea that's the confusing part, I don't know what that means yet . 

that part of the code has been the same since I gave the review

that's why I'm trying to get what it means that its no longer an issue

Reply:
it's actually fine. I think there was quite some confusion here but not a blocker